.gitignore

```diff
@@ -8,3 +8,9 @@ tmp/
 news.html
 inference_stats.png
 website/html/img-tmp
+Secrets.pm
+release-config.pm
```

Comment:
What is this release-config.pm file?

Reply:
Its the GKB::Release::Config file. If you look at the release-container branch, and in the run_release_container.sh script: 

https://github.com/reactome/Release/blob/release-container/container/run_release_container.sh#L65

It was a config file that just needed to be renamed